docs/www/faq.md

```diff
@@ -14,7 +14,7 @@ which we have tested with:
 | -------- | ------ | ---- |
 | Java | Apache Kafka Java Client | https://github.com/apache/kafka |
 | C/C++ | librdkafka | https://github.com/edenhill/librdkafka |
-| Go | Sarama | https://github.com/Shopify/sarama |
+| Go | franz-go | https://github.com/twmb/franz-go |
```

Comment:
May be worth still mentioning that we have tested with sarama, though, even though it's not as recommended?

Reply:
i think more options make folks more confused.